frontend/src/components/pages/consumers/Group.Details.tsx

```diff
@@ -128,11 +128,11 @@ class GroupDetails extends PageComponent<{ groupId: string }> {
         const totalPartitions = group.members.flatMap((m) => m.assignments).sum((a) => a.partitionIds.length);
 
         return (
-            <MotionDiv style={{ margin: '0 1rem' }} className="groupDetails">
+            <PageContent className="groupDetails">
                 {/* Statistics Card */}
                 {uiSettings.consumerGroupDetails.showStatisticsBar && (
-                    <Card className="statisticsBar">
-                        <Row>
+                    <Section py={4}>
+                        <Row className="statisticsBar">
```

Comment:
statistics bar style was moved from the container to the row, will it have the same effect?

Reply:
I just pushed a change that should keep the fade effect you have on this working as it does for now.

<img width="1386" alt="image" src="https://user-images.githubusercontent.com/523614/192907014-fb3b5318-db32-4cb0-8487-d587a9b6628f.png">